doc/whats_new.rst

```diff
@@ -37,6 +37,8 @@ API
 
     - Now it is possible to plot only a subselection of channels in :func:`mne.viz.plot_raw` by using an array for order parameter by `Jaakko Leppakangas`_
 
+    - Added :class:`mne.decoding.gsoc.XdawnTransformer` transformer to work with two dimensional data array to comply with scikit-learn pipeline. Does not support ``overlap`` case by `Asish Panda`_
```

Comment:
@kaichogami I understand that you have a gsoc file for prototyping but there is no way we merge a gsoc.py file. You need to integrated your contrib in existing files if relevant or create one. Typically it should be in decoding/xdawn.py